src/core_plugins/kibana/public/visualize/editor/styles/_editor.less

```diff
@@ -287,6 +287,17 @@
       padding: @vis-editor-agg-editor-spacing;
     }
 
+  .vis-editor-field-ui-select .ui-select-match .btn {
```

Comment:
is there a new style guide for this ? (i somehow remember i was told not to nest)